catalog-rest-service/src/main/resources/json/schema/entity/feed/thread.json

```diff
@@ -45,6 +45,10 @@
       "description": "Data asset about which this thread is created for with format <#E/{entities}/{entityName}/{field}/{fieldValue}.",
       "$ref": "../../type/basic.json#/definitions/entityLink"
     },
+    "entityId": {
+      "description": "Entity Id of the entity that the thread belongs to.",
+      "type": "string"
```

Comment:
Use the following instead - since we use it for all entity ids (it still resolves to string)

`      "$ref": "../../type/basic.json#/definitions/uuid"`